gradle.properties

```diff
@@ -4,4 +4,4 @@ group = net.fabricmc
 description = The mod loading component of Fabric
 url = https://github.com/FabricMC/fabric-loader
 
-version = 0.6.3
+version = 0.7.0
```

Comment:
We already have an 0.7 branch... Does that mean the Mixin update is moved to 0.8? What's the blocker for that?

Reply:
don't know, this version is just a placeholder

Reply:
Mixin 0.8 for loader 0.8, sounds just right

Also mixin is still preview, so it may come in 6 month or one year, not sure.

Reply:
I still think the preview version is superior to our 0.7.x fork overall. IIRC, the only blocker is not transforming the ASM namespace used in Mixin 0.7.x plugins to the Mixin 0.8.x format.